api/api.cc

```diff
@@ -300,9 +299,12 @@ future<> set_server_done(http_context& ctx) {
         rb->register_function(r, "lsa", "Log-structured allocator API");
         set_lsa(ctx, r);
 
-        rb->register_function(r, "collectd",
-                "The collectd API");
-        set_collectd(ctx, r);
+        rb->register_function(r, "commitlog",
+                "The commit log API");
+        set_commitlog(ctx,r);
+        rb->register_function(r, "error_injection",
+                "The error injection API");
+        set_error_injection(ctx, r);
```

Comment:
I don't understand what I'm seeing here... I can understand why you removed the collectd lines, buy why did you add here commitlog and error injection lines? What's the connection?

Reply:
Hmm, I see this now. I don't understand why Github asked me to review an outdated commit :-(

Reply:
I think it's my fault, in the sense that I should have fixed this in the same commit and squash it.